### DISCUSSION TITLE
Organize comics based on publisher, then series

### DISCUSSION THREAD

**Discussion by pjones112000** [Kareadita/Kavita#3352] [Ideas]
### Idea Description

For those of us that have a massive library, it would be beneficial to see in the side nav under Comics a second level of Publisher, third level of Series+Volume.  
Example:  Comics
                       DC
                       Image
                       Marvel
                              Avengers (1996)
Alternatively, a means of doing this with a Smart Filter...which unless I'm mistaken is not currently possible.
             
            

### Idea Category

User Experience

### Duration of Using Kavita

2 days

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
![msedge_gL7ShrYif2](https://github.com/user-attachments/assets/c3aa830b-913a-45db-858f-3de7904bcc45)
absolutely possible with a smart filter.

**Comment by majora2007** [comment]
Many users use Smart filters for this. I have no plans to ever support nested grouping like your describing, especially since only a small subset of users even have publisher tagged appropriately and some series have multiple publishers. I'm going to close the FR but you are more than welcome to comment after the fact.
